src/overlays/actors/ovl_En_Osn/z_en_osn.h

```diff
@@ -9,9 +9,23 @@ typedef void (*EnOsnActionFunc)(struct EnOsn*, GlobalContext*);
 
 typedef struct EnOsn {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x90];
+    /* 0x0144 */ ColliderCylinder collider;
+    /* 0X0190 */ SkelAnime anime;
```

Comment:
We prefer to name this `skelAnime` instead of just `anime`
```suggestion
    /* 0X0190 */ SkelAnime skelAnime;
```